css/style.css

```diff
@@ -0,0 +1,178 @@
+nav {
```

Comment:
`SUGGESTION`

Consider adding `margin: 0` for the body element to prevent the default margin on the page.